Gems/PhysX/Code/Source/Scene/PhysXScene.cpp

```diff
@@ -1212,15 +1217,8 @@ namespace PhysX
 
     void PhysXScene::UpdateAzProfilerDataPoints()
     {
-        using physx::PxGeometryType;
-
-        bool isProfilingActive = false;
-        if (auto profilerSystem = AZ::Debug::ProfilerSystemInterface::Get(); profilerSystem)
-        {
-            isProfilingActive = profilerSystem->IsActive();
-        }
-
-        if (!isProfilingActive)
+#if !defined(_RELEASE)
```

Comment:
Please use `AZ_RELEASE_BUILD` instead of `_RELEASE`.

Reply:
Updated, thanks! 
There are 300+ cases of using `_RELEASE` in the codebase but only 13 of `AZ_RELEASE_BUILD`..

Reply:
Yeah... hopefully they'll be replaced with time bit by bit when code changes happen around them. I always encourage in PRs to use `AZ_RELEASE_BUILD`.